apps/src/code-studio/peer_reviews/PeerReviewSubmissionData.story.jsx

```diff
@@ -11,7 +11,6 @@ export default storybook => {
         name: 'Escalated Peer Review Submissions',
```

Comment:
How are these stories different now, if both have the escalated status for all the submissions?

Reply:
🤦‍♂️ Great catch - they're not! I'll tear one out.